### PR TITLE
RD2: Sustainer Evaluation

### PR DESCRIPTION
**Pull request by andrewyu-salesforce**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/objects/Account/fields/Sustainer__c.field-meta.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Sustainer__c</fullName>
+    <description>Determine if the Account is a sustainer</description>
```
Comment (andrewyu-salesforce) [inline force-app/main/default/objects/Account/fields/Sustainer__c.field-meta.xml:4]: @sam-knox Can you provide the description for Account/Contact Sustainer field?

Comment (sam-knox) [inline force-app/main/default/objects/Account/fields/Sustainer__c.field-meta.xml:4]:
```suggestion
    <description>Indicates if this Account is an active, lapsed, or former recurring donor. The value is based on the Status of the related Recurring Donations.</description>
```

File: force-app/main/default/objects/Account/fields/Sustainer__c.field-meta.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <externalId>false</externalId>
+    <inlineHelpText>Determine if the Account is a sustainer</inlineHelpText>
```
Comment (sam-knox) [inline force-app/main/default/objects/Account/fields/Sustainer__c.field-meta.xml:6]:
```suggestion
    <inlineHelpText>Indicates if this Account is an active, lapsed, or former recurring donor. The value is based on the Status of the related Recurring Donations.</inlineHelpText>
```

File: force-app/main/default/objects/Contact/fields/Sustainer__c.field-meta.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Sustainer__c</fullName>
+    <description>Determine if the Contact is a sustainer</description>
```
Comment (sam-knox) [inline force-app/main/default/objects/Contact/fields/Sustainer__c.field-meta.xml:4]:
```suggestion
    <description>Indicates if this Contact is an active, lapsed, or former recurring donor. The value is based on the Status of the related Recurring Donations.</description>
```

File: force-app/main/default/objects/Contact/fields/Sustainer__c.field-meta.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    <externalId>false</externalId>
+    <inlineHelpText>Determine if the Contact is a sustainer</inlineHelpText>
```
Comment (sam-knox) [inline force-app/main/default/objects/Contact/fields/Sustainer__c.field-meta.xml:6]:
```suggestion
    <inlineHelpText>Indicates if this Contact is an active, lapsed, or former recurring donor. The value is based on the Status of the related Recurring Donations.</inlineHelpText>
```

**Review by sam-knox — CHANGES_REQUESTED** [review]
Ok, I just added the description and help text for these fields.

**Pull request closed without merging**